chemprop/v2/cli/train.py

```diff
@@ -615,16 +628,17 @@ def main(args):
         test_loader = None
 
     model = MPNN(
-        mp_block,
-        agg,
-        readout_ffn,
-        None,
-        args.task_weights,
-        args.warmup_epochs,
-        args.num_lrs,
```

Comment:
remove `args.num_lrs` if it's not in the `MPNN.__init__` signature.

Reply:
`num_lrs` is currently commented out in `MPNN.__init__`. Do you know we will implement it in the future? If not, I can clean up the `MPNN.__init__` as well.

Reply:
We have decided to not implement it in the future. Thank you Xiaorui.